docs/04-guides/on-demand/store-video-asset-on-ipfs/01-dashboard.mdx

```diff
@@ -0,0 +1,32 @@
+---
+sidebar_class_name: tabbed-page
+hide_title: true
+slug: dashboard
+title: Dashboard
+pagination_next: guides/on-demand/upload-a-video-asset
+---
+
+import Tabs from "./_tabs.mdx";
+
+<Tabs />
+
+The Livepeer Studio Dashboard is a frontend interface that enables users to interact 
+with Livepeer’s features without any code. Using the Dashboard you can upload and modify a 
```

Comment:
"The Livepeer Studio Dashboard is a frontend interface that enables ~users~ you to interact 
with Livepeer’s ~features~ API without any code. Using the Dashboard you can ~upload and modify a video asset quickly just by uploading it to Livepeer~ you can replicate uploaded video assets to additional storage providers such as IPFS.